gapii/cc/gles_mid_execution.cpp

```diff
@@ -975,8 +975,8 @@ ImageData Reader::ReadRenderbuffer(Renderbuffer* rb) {
       "MEC: Reading renderbuffer %d format 0x%x type 0x%x sized 0x%x at 0x%x",
       rb->mID, img->mDataFormat, img->mDataType, format, attach);
   if (attach == GL_COLOR_ATTACHMENT0) {
-    auto readFb = CreateAndBindFramebuffer(GL_READ_FRAMEBUFFER);
-    imports.glFramebufferRenderbuffer(GL_READ_FRAMEBUFFER, GL_COLOR_ATTACHMENT0,
+    auto readFb = CreateAndBindFramebuffer(GL_FRAMEBUFFER);
```

Comment:
Please add a comment explaining that this is a workaround for a potential driver bug. It looks like this is a mistake with your code, and someone might try to fix it.

Reply:
Done.